spec/extensions/fourier_transform_functions.md

```diff
@@ -0,0 +1,505 @@
+# Fourier transform Functions
+
+> Array API specification for Fourier transform functions.
+
+A conforming implementation of the array API standard must provide and support the following functions adhering to the following conventions.
+
+-   Positional parameters must be [positional-only](https://www.python.org/dev/peps/pep-0570/) parameters. Positional-only parameters have no externally-usable name. When a function accepting positional-only parameters is called, positional arguments are mapped to these parameters based solely on their order.
+-   Optional parameters must be [keyword-only](https://www.python.org/dev/peps/pep-3102/) arguments.
+-   Broadcasting semantics must follow the semantics defined in {ref}`broadcasting`.
+-   Unless stated otherwise, functions must support the data types defined in {ref}`data-types`.
+-   Unless stated otherwise, functions must adhere to the type promotion rules defined in {ref}`type-promotion`.
+-   Unless stated otherwise, floating-point operations must adhere to IEEE 754-2019.
+
+## Objects in API
+
+<!-- NOTE: please keep the functions and their inverse together -->
+(function-fft)=
+### fft(a, /, *, n=None, axis=-1, norm='backward')
+
+Computes the one-dimensional discrete Fourier transform. The expected behavior includes a round-trip transform using the inverse function, `fft(ifft(a)) == a`.
```

Comment:
It's best to revert the order. For C2C transforms it doesn't matter, but it matters for the R2C->C2R round trip.
```suggestion
Computes the one-dimensional discrete Fourier transform. The expected behavior includes a round-trip transform using the inverse function, `ifft(fft(a)) == a`.
```